misago/markup/tests/test_parser.py

```diff
@@ -192,7 +192,7 @@ def test_clean_current_link(self):
 """.strip()
 
         expected_result = """
-<p>Lorem ipsum: <a href="/" rel="nofollow">test.com</a></p>
```

Comment:
I think we have not add rel="nofollow" to internal links (they are not decrease search engines rating, see for ex. https://www.youtube.com/watch?v=86GHCVRReJs).
External links (to any other domain) are still `nofollow`